modules/core/shared/src/main/scala/lucuma/core/model/EphemerisTracking.scala

```diff
@@ -18,8 +18,8 @@ import scala.collection.immutable.TreeMap
  * Time-parameterized coordinates over a fixed interval, defined pairwise. Coordinates that fall
  * between known instants are interpolated.
  */
-sealed abstract case class Ephemeris private (toMap: TreeMap[Timestamp, EphemerisCoordinates]) {
-  import Ephemeris.Element
+sealed abstract case class EphemerisTracking private (toMap: TreeMap[Timestamp, EphemerisCoordinates]) {
```

Comment:
Would this not also implement `Tracking`?

Reply:
Well that would make sense, yes. Thanks.

Reply:
yes, we use tracking extensively on the visualization and explore in general